scripts/setup_fvp_optee.sh

```diff
@@ -216,7 +218,8 @@ cat >> $DST_GEN_ROOTFS/filelist-tee.txt << EOF
 # OP-TEE device
 dir /lib/modules 755 0 0
 dir /lib/modules/$KERNEL_VERSION 755 0 0
-file /lib/modules/$KERNEL_VERSION/optee.ko $DST_OPTEE_LK/optee.ko 755 0 0
+file /lib/modules/$KERNEL_VERSION/optee.ko $DST_OPTEE_LK/core/optee.ko 755 0 0
+file /lib/modules/$KERNEL_VERSION/optee_armtz.ko $DST_OPTEE_LK/armtz/optee_armtz.ko 755 0 0
```

Comment:
Should be fine. It has been already done on "my" setup_juno_optee.sh. After discussion with Pascal, I will create a PR for the setup_juno_optee.sh script once the linuxdriver type II and dependencies are merged.